pkg/cmd/artefactor.go

```diff
@@ -23,8 +23,13 @@ const (
 	FlagWebFiles = "web-files"
 	// FlagLogs enabled debug logs
 	FlagLogs = "logs"
+	// FlagTargetPlatform allows the correct version of artefactor to be saved
+	// with files.
+	FlagTargetPlatform = "target-platform"
 	// DefaultArchiveDir
 	DefaultArchiveDir = "downloads"
+	// The defualt binary type to include in downloads
```

Comment:
s/defualt/default/ .. and for a sake of a linter add `DefaultTargetPlatform` start of comment

Reply:
Added checks in build and fixed this